src/coverlet.core/CoverageResult.cs

```diff
@@ -196,33 +196,26 @@ public ThresholdTypeFlags GetThresholdTypesBelowThreshold(CoverageSummary summar
                     break;
                 case ThresholdStatistic.Average:
                     {
-                        double line = 0;
-                        double branch = 0;
-                        double method = 0;
+                        var line = summary.CalculateLineCoverage(Modules).AverageModulePercent;
```

Comment:
nit: `double`

Reply:
Done... just one thing: lines 173 to 175 & 225 to 227 are using `var`, LMK if var o double is better, I can do a quick push if needed

Reply:
yes please push fast update to keep style.

Reply:
The rule I follow is "if the type is clear from assignment it's ok var but if it's not clear better make clean".

Reply:
Done!